lib/sequenceserver.rb

```diff
@@ -138,10 +138,13 @@ def on_stop
       puts
       puts '** Thank you for using SequenceServer :).'
       puts '   Please cite: '
-      puts '       Priyam A, Woodcroft BJ, Rai V, Munagala A, Moghul I, Ter F,'
-      puts '       Gibbins MA, Moon H, Leonard G, Rumpf W & Wurm Y. 2015.'
-      puts '       Sequenceserver: a modern graphical user interface for'
-      puts '       custom BLAST databases. biorxiv doi: 10.1101/033142.'
+      puts '       Priyam A, Woodcroft BJ, Rai V, Moghul I, Munagala A, Ter F,'
+      puts '       Chowdhary H, Pieniak I, Maynard LJ, Gibbins MA, Moon H,'
+      puts '       Davis-Richardson A, Uludag M, Watson-Haigh N, Challis R,'
+      puts '       Nakamura H, Favreau E, Gómez EA, Pluskal T, Leonard G,'
+      puts '       Rumpf W & Wurm Y. 2019.'
+      puts '       Sequenceserver: A Modern Graphical User Interface for'
+      puts '       Custom BLAST Databases. MBE doi: 10.1101/033142.'
```

Comment:
I think I would prefer the full name of the journal. FYI, if you are abbreviating journal names you should use the standardised (ISO4) abbreviations.

Reply:
ok